runner/app/pipelines/text_to_image.py

```diff
@@ -94,6 +94,8 @@ def __init__(self, model_id: str):
             self.ldm = AutoPipelineForText2Image.from_pretrained(model_id, **kwargs).to(
                 torch_device
             )
+        if os.environ.get("LOWVRAM"):
+            self.ldm.enable_sequential_cpu_offload()
```

Comment:
@Titan-Node Have you tried [model offloading](https://huggingface.co/docs/diffusers/main/en/optimization/memory#model-offloading) instead of sequential CPU offloading? Would be curious how the VRAM savings + inference speed results compare given that model offloading is supposed to have less of an impact on inference speed.

Reply:
Interestingly with model offloading it ended up increasing the GPU memory requirements. I tried both with torch_device being set to GPU and CPU but model offloading had either no effect or increased GPU RAM while also slowing down inference by 4x. 
I've also tried model offloading with text, image and video pipelines with the same results, I gave up on it once I seen the sequential offloading option. I tried both together but I believe it throw an error. 

Here is some breakdowns of tests I did.
[benchmarks.xlsx](https://github.com/livepeer/ai-worker/files/14607165/benchmarks.xlsx)